generators/client/templates/react/src/main/webapp/app/config/dayjs.ts.ejs

```diff
@@ -0,0 +1,11 @@
+import dayjs from 'dayjs'
```

Comment:
I don't know if it's better to add semicolon at the end of the lines 🤔

Reply:
Yes, I think that the convention in JHipster is to use it, you're totally right to notice this, I just added it where necessary